Makefile

```diff
@@ -173,6 +181,21 @@ _do_e2e_test:
 	CGO_ENABLED=0 go test -v -c -o bin/devworkspace-controller-e2e ./test/e2e/cmd/workspaces_test.go
 	./bin/devworkspace-controller-e2e
 
+# it's easier to bump whole kubeconfig instead of grabbing cluster URL from the current context
+_bump_kubeconfig:
+	mkdir -p $(BUMPED_KUBECONFIG_FLD)
+ifndef KUBECONFIG
+	$(eval CONFIG_FILE = ${HOME}/.kube/config)
+else
+	$(eval CONFIG_FILE = ${KUBECONFIG})
+endif
+	cp $(CONFIG_FILE) $(BUMPED_KUBECONFIG)
+
+_login_with_devworkspace_sa:
+	$(eval SA_TOKEN := $(shell $(TOOL) get secrets -o=json -n $(NAMESPACE) | jq -r '[.items[] | select (.type == "kubernetes.io/service-account-token" and .metadata.annotations."kubernetes.io/service-account.name" == "devworkspace-controller")][0].data.token' | base64 --decode ))
+	@echo "Logging as devworkspace controller SA"
+	@oc login --token=$(SA_TOKEN) --kubeconfig=$(BUMPED_KUBECONFIG)
```

Comment:
`@` suppressed printing a command before execution but it's printed when `make plan` is printed. If you know how to avoid it - contribute or let me know
![Screenshot_20200731_123636](https://user-images.githubusercontent.com/5887312/89022351-8fa0f900-d32a-11ea-97e4-16d8ed1fbfba.png)

Reply:
Since we have `.ONESHELL` enabled, it only checks for an `@` at the start of the first line in the rule. I'll fix it up.